app/main.c

```
@@ -203,6 +205,9 @@ static char ble_adv_name[20] = {0};
 
 static uint8_t bat_level_flag = 0;
 
+static uint8_t pending_passkey[PASSKEY_LENGTH] = {0};
```

Clang (14.0.6)

[warning] variable 'pending_passkey' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)